Tests/conf.json

```diff
@@ -3162,6 +3162,10 @@
             "integrations": "Public DNS Feed",
             "playbookID": "Public_DNS_Feed_Test",
             "fromversion": "5.5.0"
+        },
+        {
+            "integrations": "CyrenThreatInDepth",
+            "playbookID": "CyrenThreatInDepth-Test"
```

Comment:
We don't have access to this instance, so this test will always fail in our build.
Because of that, it should be removed from `conf.json`.
```suggestion
```

Reply:
What do you mean? We were told to provide you with API tokens so that this can run on a nightly basis. Is this the wrong place to register this?

Reply:
That request is probably out dated.
We do not run partner packs as part of the XSOAR nightly build.
With that being said, sharing instance creds with us will allow us to assist in case of an ad-hoc customer request or bug, but it is not mandatory.

Reply:
I see, will remove